includes/html/pages/device/port.inc.php

```diff
@@ -6,32 +6,31 @@
 
 $vars['view'] = basename($vars['view'] ?? 'graphs');
 
-$port = dbFetchRow('SELECT * FROM `ports` WHERE `port_id` = ?', [$vars['port']]);
+$port = \App\Models\Port::find($vars['port']);
```

Comment:
This will not work since it gets inherited into all `includes/html/pages/device/port/*.inc.php` files ?